lib/src/laminate/popup/src/popup_size_provider.dart

```diff
@@ -39,3 +39,22 @@ class PercentagePopupSizeProvider implements PopupSizeProvider {
   num getMaxHeight(num _, num viewportHeight) =>
       viewportHeight * _maxHeightPercentage;
 }
+
+/// Provides a popup size with a fixed maximum width and height in pixels.
+///
+/// A null value for max width or max height indicates no limit.
+class FixedPopupSizeProvider implements PopupSizeProvider {
```

Comment:
Isn't that name misleading? 
The name says `FixedPopupSize` but it seems to be a `MaxSizePopupSizeProvider`. 
What if you want to add a toaster component and need a popup with fixed size for that?